roles/monitoring_server/templates/prometheus_rules.yml.j2

```diff
@@ -13,44 +13,44 @@ groups:
     rules:
       - alert: EndpointDown
         expr: probe_success == 0
-        for: 1m
+        for: 2m
         labels:
           severity: critical
         annotations:
           summary: "Endpoint {{ $labels.instance }} down"
 
       - alert: InstanceDown
         expr: up == 0
-        for: 5m
+        for: 10m
         labels:
           severity: high
         annotations:
           summary: "{{ $labels.job }} instance {{ $labels.instance }} down"
-          description: "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 5 minutes."
+          description: "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 10 minutes."
 
   - name: metrics
     rules:
       - alert: MemoryUsageHigh
         expr: 100- node_memory_MemAvailable_bytes/node_memory_MemTotal_bytes*100 > 90
-        for: 5m
+        for: 10m
```

Comment:
I think this should stay as 5m - although we're getting a lot of alerts it has only been from instances which are under memory pressure (i.e. they are under spec'd). We should silence the alerts when we are waiting for a memory increase rather than increasing the time here.